lib/config/config.js

```diff
@@ -234,6 +246,35 @@ exports.setDefaults = function (sourcePath) {
 
 
 /**
+ * Dynamically set a configuration value
+ *
+ * @param {String} name       The key to assign the content to.
+ * @param {*} value 		  Value
+ */
+exports.set = function (name, value) {
+	const obj = {};
+	const subpaths = name.split('.');
```

Comment:
In `exports.get`, the name (ie: `trail`) is allowed to be an array I think. Might be nice to support that in `set` too.
Also, `trail` would be a more consistent name than `name` given the rest of this module.